### DISCUSSION TITLE
Fraction unbound (plasma, reference value) vs Fraction unbound (experiment)

### DISCUSSION THREAD

**Discussion by Xiaoyang-svg** [Open-Systems-Pharmacology/Forum#1452] [Questions & Problems]
Dear community, 

If I only consider the free fraction of the drug in the plasma, I wonder whether the values of fraction unbound (plasma, reference value) and fraction unbound (experiment) have to be the same？
![image](https://user-images.githubusercontent.com/102672543/218254295-09cadf9c-b9f7-4ee6-a639-26cdf9893602.png)

Looking for your reply.

Thank you!!

**Comment by PavelBal** [comment]
The `Fraction unbound (experiment)` is used for calculation of the specific renal clearance rate constant from renal clearance input value. So to correctly apply the measured or predicte renal clearance value to the model, the two `Fraction unbound` parameters should be equal. You should also consider this during parameter identification, if you are optimizing the fraction unbound.

**Reply by Xiaoyang-svg** [reply]
Thank you!
